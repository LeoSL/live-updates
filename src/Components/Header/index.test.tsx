import { render } from "@testing-library/react";
import { Header } from ".";

it("renders correctly", () => {
  const { container } = render(<Header />);
  expect(container).toMatchSnapshot();
});
