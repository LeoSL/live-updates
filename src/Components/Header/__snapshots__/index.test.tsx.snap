// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-3"
  >
    <img
      alt="spinning logo"
      class="makeStyles-appLogo-2"
      src="logo.svg"
    />
  </div>
</div>
`;
